deployment/data/nginx/app.conf

```diff
@@ -0,0 +1,62 @@
+upstream core_backend {
+    server core_backend:8000 fail_timeout=0;
+}
+
+upstream admin_app {
+    server admin_app:3000 fail_timeout=0;
+}
+
+server {
+    listen 80;
```

Comment:
Note that we we haven't closed the unencrypted HTTP on port 80.
Once we have ironed this out, I'll raise another PR to redirect all port 80 to 443.